cmd/docker.go

```diff
@@ -48,7 +51,15 @@ func (d *KoolDocker) Execute(args []string) (err error) {
 	image := args[0]
 	workDir, _ := os.Getwd()
 
-	if !d.Flags.DisableTty && !environment.IsTrue("KOOL_TTY_DISABLE") {
+	if d.Flags.DisableTty {
+		d.Warning("Warning: --disable-tty flag is obsolete")
```

Comment:
I guess we should not place the warning because some usage relies on something like `kool exec foo bar > xxx.sql` and having the warning will make that break... 

we could if d.Warning wrote to `Stderr` instead of straight `Stdout`... I guess removing the warning is safer - as we expect the automatic behaviour to be best than people asserting themselves whether or not to use TTY flags.